src/ukify/test/example.tpm2-pcr-private2.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQC2v6Mhdx7kucPr
+nkE4R+cqgWf9Opushv3j7Hnt+O0v/8wivOPE59K0v/EbN8oxL6DYCWSBBEN/DBy2
+LQ60nWRtpY7bnlG+pKUy4oH4MevBGbjeHkjOKSwMaUV4k8ReRJ88qVuSS1JuNEmM
+gyHDE4aO4ngtnTPVYw52uPHpmk7A8TWW7iKdNIYfV8+nGZD5r3EiezDlQCNDnxRG
+vnnHVzTXYGtpclZyiIrZUzJA4QOftnypyP5kA/x5S51OPxaqZP7xkOKCbqE/ffo0
+EN/kNz47FhPe1mPGRFYZWGex4iaOtyKtxgSQXbGe4EhyTxJ2eOu8AEhVISv8zQOg
+SmmlvPd/AgMBAAECggEAQAiDDQFTwlozA5a2jJVpMvQX71t/W6MLSDc+e/tqhJSR
+PyTHfGGsa2gL//j68GQbbEdSP4Cyc8xXLSA5lGDH5UGK/noJc472UY+bcc9wJ3+v
+Tqj24sH7bLfgP0Ern8pUr2k6LFcXIYVRthnmlRd84QkKihUYqe7lDQV9wlguuxzQ
+pfTKCMi5mreb218LGKD+1LcVeXf11jg7gbg2YKgWNCdwVb2S2yWHSN0eOxOwmdYr
+IEBzJFxG60RqJRuG5Hjobzg6s/rqJ5LqmkraQhzlqOAVKnZF8ziloop8WQpPctMs
+tppcs8mbAdXzhI5c7E5UZC3cIpGzHN+h6K+AwGvDyQKBgQD4A97P3oothh0vGBaV
+vV9xGNmXmnSyH4ooGrbohmIZI0VXYvk9uebIBcl6Q1LxZswPZQ1XyMHiN65gA5zh
+j/Gdg+t9oqNBfitMAkQ9uiloiyJUhXnY9DC/F+efK2pJMlwdr/jXKmDzdAFAT82Y
+VFbw2ZKV/F4BM1KBt0Y7DONiZwKBgQC8odfM0k+j/nUK48LEv0kFmSLuggMYd3xU
+fbnltqHEMZIeN98UG+hAXGpuMXkBO3c0rnYD5WfCAG1qOUve6stwz7En+xVvY/qa
+SvSh4s38gfPHyxQhbo5dpA6TOzpOC2V/kUpUEGIJeUVYaCNnYsiR4VPeV/YoGXmK
+PWoJnp0DKQKBgBY7qpax2Ws5UZZuL2AfCNZHpwHrK7joEOeFdY4ktdiRC99IlRVO
+e/zEYApgzKetUm+y3F5ZNeBEo5Ih4Mdrsvou4EZz9PSjDdiTf3CVJq8jgeFX0dN8
+GH6Y8v+W0cFcDVvv8XvAZc6NQKt2O/UC4oRWzMg7RmYPJr9QIbCbeCrTAoGAN7Yl
+Il1LIEhbEShLsgg87OZZps/HUkaX9excJzhVdridS0d9H1dOtROWa405C+AgSPLt
+8CClEGtH5YOeoOv/wgXVcNV7cza4ITHaJqXx6I4JDg0wmN8qNQXrOBjaE4rSI2cp
+6MId8mZa0M2RB0vpqQw/1P9tuFItz2JsGwM5pGECgYAUgAUwXCApKYVFEFlG6Pac
+ou0aw7F6mZ2/M5G/zOm0qCbsWtaMSOSuA/6iU9pt40ZYAN4U/wfqnw2VEhFp7w1M
+zYZbBD0qeYdp4fsSnYqLfbAVlP/0zva3vL02REknVjPUJp/hjJUhA+mV7nvT6yV4
+q88Ieo8LwCW5sbmwirSpmw==
+-----END PRIVATE KEY-----
```

Comment:
I've already received a bunch of reports from automatic scanners maintained by RH. I was actually impressed that they _immediately_ flagged something pushed to a branch. As a result of that, I renamed the files to `example.*`. I added base64 obfuscation now too.

Reply:
I am pretty sure more low-quality scanners will happily ignore the "test" and "example" prefixes and just barf anyway... Generating on the fly with openssl is not too difficult, any reason not to just do that?

Reply:
openssl command wasn't used at all, so that'd pull that in, and require some code to generate the file. And the generation of the file would probably take longer than all the other tests combined, because openssl would gather "true" entropy.